selfdrive/modeld/modeld.cc

```diff
@@ -138,10 +138,9 @@ void run_model(ModelState &model, VisionIpcClient &vipc_client_main, VisionIpcCl
     }
 
     // Enable/disable nav features
-    double tsm = (float)(nanos_since_boot() - sm["mapRenderState"].getMapRenderState().getLocationMonoTime()) / 1e6;
-    bool route_valid = sm["navInstruction"].getValid() && (tsm < 1000);
-    bool render_valid = sm["mapRenderState"].getValid() && (sm["navModel"].getNavModel().getFrameId() == sm["mapRenderState"].getMapRenderState().getFrameId());
-    bool nav_valid = route_valid && render_valid;
+    uint64_t timestamp_llk = sm["navModel"].getNavModel().getLocationMonoTime();
+    double tsm = (float)(nanos_since_boot() - timestamp_llk) / 1e6;
+    bool nav_valid = sm["navInstruction"].getValid() && sm["navModel"].getValid() && (tsm < 1000);
```

Comment:
also 1ms seems too strict?

Reply:
I think the threshold is 1000ms here?

Reply:
hmm, that sounds like too much, but this is in nanoseconds. if it's not clear from the code, we should rename so it is

Reply:
I see 1s is two navModel packets